src/json-formatter.less

```diff
@@ -76,6 +76,16 @@
       transform: rotate(90deg);
     }
   }
+
+  >a>.thumbnail-text {
+    opacity: 0;
+    transition: .15s;
```

Comment:
You are transitioning all properties. Please specify transition property to be only opacity. Also use `ease-in` transition function for a nicer effect.